coral-service/src/main/java/com/linkedin/coral/coralservice/controller/VisualizationController.java

```diff
@@ -43,9 +44,9 @@ public ResponseEntity getIRVisualizations(@RequestBody VisualizationRequestBody
     final String query = visualizationRequestBody.getQuery();
     final RewriteType rewriteType = visualizationRequestBody.getRewriteType();
 
-    if (!visualizationUtils.isValidSourceLanguage(sourceLanguage)) {
+    if (!isValidSourceLanguage(sourceLanguage)) {
       return ResponseEntity.status(HttpStatus.BAD_REQUEST)
-          .body("Currently, only Hive, Spark, and Trino are supported as engines to generate graphs using.\n");
+          .body("Currently, only Hive, and Trino are supported as engines to generate graphs using.\n");
```

Comment:
Let us allow Spark and route it to Hive. Also you can say
`Currently, only Hive, Spark, and Trino SQL are supported as source languages for Coral IR visualization`

Reply:
Do we also do this for the translation endpoint?